packages/language-support/package.json

```diff
@@ -48,7 +48,7 @@
     "vscode-languageserver-types": "^3.17.3"
   },
   "scripts": {
-    "gen-parser": "antlr4 -Dlanguage=TypeScript -visitor src/antlr-grammar/CypherCmdLexer.g4 src/antlr-grammar/CypherCmdParser.g4 -o src/generated-parser/ -Xexact-output-dir",
+    "gen-parser": "export ANTLR4_TOOLS_ANTLR_VERSION=4.13.2 && antlr4 -Dlanguage=TypeScript -visitor src/antlr-grammar/CypherCmdLexer.g4 src/antlr-grammar/CypherCmdParser.g4 -o src/generated-parser/ -Xexact-output-dir",
```

Comment:
This is because https://github.com/antlr/antlr4-tools will look for the most up to date version of antlr otherwise. But it was failing contacting the relevant url, so this hardcodes the version we want to use (which we should anyway because we are using 4.13.2 and we wouldn't want to use a more recent one for generating the code but an older one as runtime).